cmd/launcher/main.go

```diff
@@ -71,13 +78,16 @@ func main() {
 		if err := runSubcommands(systemSlogger); err != nil {
 			logutil.Fatal(logger, "err", fmt.Errorf("run with positional args: %w", err))
```

Comment:
should probably remove all the `logutil.Fatal()` as well since they call `os.Exit(1)`

Reply:
Oh yup absolutely, thank you!